app/src/main/res/values/styles.xml

```diff
@@ -7,5 +7,7 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
-
```

Comment:
why remove this line? There could be a blank line after each style.

Reply:
okay I reverted that change.